ftplugin/neoman.vim

```diff
@@ -3,31 +3,24 @@ if exists('b:did_ftplugin')
 endif
 let b:did_ftplugin = 1
 
-setlocal iskeyword+=\.,-,(,),:
-setlocal buftype=nofile
-setlocal noswapfile
-setlocal nofoldenable
-setlocal bufhidden=hide
-setlocal nobuflisted
-setlocal nomodified
-setlocal readonly
-setlocal nomodifiable
-setlocal noexpandtab
-setlocal tabstop=8
-setlocal softtabstop=8
-setlocal shiftwidth=8
-setlocal nolist
-setlocal foldcolumn=0
-setlocal colorcolumn=0
+setl buftype=nofile
+setl noswapfile
+setl nofoldenable
+setl bufhidden=hide
+setl nobuflisted
+setl nomodified
+setl readonly
+setl nomodifiable
+setl noexpandtab
+setl tabstop=8
+setl softtabstop=8
+setl shiftwidth=8
+setl nolist
+setl foldcolumn=0
+setl colorcolumn=0
```

Comment:
Abbreviated names in scripts are generally advised against. Abbreviation feature of Vim is for human input (i.e. at the `:` prompt).